pkg/config/autodiscover/helm_info.go

```diff
@@ -0,0 +1,22 @@
+package autodiscover
+
+type HelmSetList struct {
+	Items []HelmList `json:""`
+}
+type HelmList struct {
+	Name      string `json:"name"`
+	Namespace string `json:"namespace"`
+	Chart     string `json:"chart"`
```

Comment:
I think ```HelmList ```  is not an appropriate name. This structure does not contain any list, it defined a single Helm object.